clientlib/src/main/proto/yelp/nrtsearch/search.proto

```diff
@@ -766,6 +766,7 @@ message SearchResponse {
         double rescoreTimeMs = 10;
         map<string, double> rescorersTimeMs = 11;
         map<string, Diagnostics> innerHitsDiagnostics = 12;
+        double loggingHitsTimeMs = 13;
```

Comment:
This needs to use the same field number as the main branch, to keep the clientlib compatible

Reply:
thanks for pointing this out!